src/IssueViz.Security/Hotspots/HotspotsStore.cs

```diff
@@ -97,25 +80,11 @@ private IAnalysisIssueVisualization FindExisting(IAnalysisIssueVisualization hot
             return Hotspots.FirstOrDefault(x => ((IHotspot)x.Issue).HotspotKey == key);
         }
 
-        ReadOnlyObservableCollection<IAnalysisIssueVisualization> IHotspotsStore.GetAll()
-        {
-            return new ReadOnlyObservableCollection<IAnalysisIssueVisualization>(Hotspots);
-        }
-
-        public event EventHandler<IssuesChangedEventArgs> IssuesChanged;
-
-        IEnumerable<IAnalysisIssueLocationVisualization> IIssueLocationStore.GetLocations(string filePath)
-        {
-            var matchingLocations = Hotspots
-                .SelectMany(hotspotViz => hotspotViz.GetAllLocations())
-                .Where(locationViz => !string.IsNullOrEmpty(locationViz.CurrentFilePath) &&
-                                      PathHelper.IsMatchingPath(locationViz.CurrentFilePath, filePath));
-
-            return matchingLocations;
-        }
+        public ReadOnlyObservableCollection<IAnalysisIssueVisualization> GetAll() => ReadOnlyWrapper;
```

Comment:
I wasn't very comfortable with the way the old code return a new wrapper for every call - this feels nicer.